### DISCUSSION TITLE
Release notes along w/ github releases

### DISCUSSION THREAD

**Discussion by rabbiveesh** [fief-dev/fief#9] [Ideas]
I like to track projects via subscribing to releases on github. Is it possible to include release notes along with github releases?

**Comment by frankie567** [comment]
Yeah, actually I add release notes to the releases but *after* it has been created. The problem is the release is automatically created in the CI and I edit after to add the details. I guess you get the notification only when it's created, right?

I'll think about a better way to handle this!

**Comment by rabbiveesh** [comment]
Ah, thanks!

**Reply by frankie567** [reply]
I've finally managed to have a way of publishing the release notes properly in the GitHub release :) You should have seen it in action for the latest version, [v0.14.0](https://github.com/fief-dev/fief/releases/tag/v0.14.0).

Cheers!
